panels/templates/panel_app_form.html

```diff
@@ -9,6 +9,9 @@
             isNotAnHour = $.val('length') != {{ c.SIXTY_MIN }};
         setVisible($('#other-length'), isOther);
         $.field('length_text').prop('required', isOther);
+        if (!isOther) {
```

Comment:
I think all the data on labs/west is how it's supposed to be, so we should be safe. But I'll double check.